docs/src/getting_started.ipynb

```diff
@@ -8,7 +8,17 @@
    "source": [
     "# Getting Started with Temporian\n",
```

Comment:
Change to `Getting Started` since the H1 is whats shown in the navbar and its too long as-is
<img width="355" alt="image" src="https://github.com/google/temporian/assets/49082859/a9b029b0-d187-4272-b824-00d14364f4b8">